BlueDBOnDisk/src/main/java/org/bluedb/disk/BlueCollectionOnDiskBuilder.java

```diff
@@ -28,16 +29,18 @@ protected BlueCollectionOnDiskBuilder(BlueDbOnDisk db, Class<? extends BlueKey>
 		return this;
 	}
 
-	@SuppressWarnings("unchecked")
 	@Override
-	public BlueCollectionOnDiskBuilder<T> withRegisteredClasses(Class<? extends Serializable>... additionalRegisteredClasses) {
-		this.additionalRegisteredClasses = additionalRegisteredClasses;
+	public BlueCollectionOnDiskBuilder<T> usingClasses(Collection<Class<? extends Serializable>> classesToRegister) {

```

Comment:
I'm not sure how I feel about the `usingClasses` name. That doesn't really communicate why it might be important to the user to call this. `register` doesn't really either though...

Reply:
I'd be happy to use a better name if you can think of one.

Reply:
rename to `withOptimizedClasses`